src/components/ratings.tsx

```diff
@@ -3,33 +3,19 @@ import { connect } from 'react-redux';
 import Ratings from 'react-ratings-declarative';
 
 import { updateBeer } from '../actions/index';
+import { IRatingsProps } from '../interfaces';
 
-class RatingsComponent extends Component {
-  getRating = () => {
-    return this.props.beerRating;
+class RatingsComponent extends Component<IRatingsProps> {
+  public state = {
```

Comment:
Add type for state. It will be second generic type in `Component<IRatingsProps>`